css-animations-2/Overview.bs

```diff
@@ -690,7 +693,173 @@ The 'animation' shorthand property syntax is as follows:
 
 <span class=prod><dfn>&lt;single-animation></dfn> = <<'animation-duration'>> || <<easing-function>> || <<'animation-delay'>> || <<single-animation-iteration-count>> || <<single-animation-direction>> || <<single-animation-fill-mode>> || <<single-animation-play-state>> || [ none | <<keyframes-name>> ] || <<single-animation-timeline>></span>
 
+<h2 id="animation-triggers">
+Declaring Animation Triggers</h2>
 
+    Animation Triggers are defined using the 'animation-trigger-*' properties.
+    These list-valued properties,
+    which are all [=longhands=] of the 'animation-trigger' [=shorthand=],
+    form a [=coordinating list property group=]
+    with 'animation-name' as the [=coordinating list base property=]
+    and each item in the [=coordinated value list=]
+    defining the properties of a single animation trigger.
+
+## The 'animation-trigger-type' property ## {#animation-trigger-type}
+
+The 'animation-trigger-type' property specifies the [=animation trigger type=]
+of the [=animation trigger=].
+
+<pre class='propdef'>
+Name: animation-trigger-type
+Value: <<single-animation-trigger-type>>#
+Initial: once
+Applies to: all elements
+Inherited: no
+Percentages: N/A
+Computed value: list, each item a keyword as specified
+Animation type: not animatable
+Canonical order: per grammar
+</pre>
+
+<span class=prod><dfn>&lt;single-animation-trigger-type></dfn> = once | repeat | alternate | state</span>
```

Comment:
I was pondering all the `none`/`normal`/`auto` options, and I think we currently don't need them, since the initial value is `once`, with a timeline value of `auto`, which I think is exactly what we have today.
We could also add an issue on that, in case it needs an approval from the group.

Reply:
The definition of the values is done [here](https://github.com/w3c/csswg-drafts/pull/10128/files#diff-d4bbb6f14218df63c7727c884dba84e0103d39de0571ef1f7ab202c0a30e7608R2434).
Should I add here another description? More user/less implementor facing?

Reply:
> Can you add a description of what each property value means.

Done